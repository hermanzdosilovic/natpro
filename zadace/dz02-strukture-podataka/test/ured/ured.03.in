100
7 49 73 58 30 72 44 78 23 9 40 65 92 42 87 3 27 29 40 12 3 69 9 57 60 33 99 78 16 35 97 26 12 67 10 33 79 49 79 21 67 72 93 36 85 45 28 91 94 57 1 53 8 44 68 90 24 96 30 3 22 66 49 24 1 53 77 8 28 33 98 81 35 13 65 14 63 36 25 69 15 94 29 1 17 95 5 4 51 98 88 23 5 82 52 66 16 37 38 44
100
sef 98 100
promjena 38 58
sef 98 98
promjena 10 31
sef 76 86
promjena 43 99
promjena 13 60
sef 95 99
sef 69 78
promjena 7 62
promjena 66 82
promjena 68 21
sef 13 36
sef 91 92
promjena 58 16
promjena 41 79
sef 7 31
promjena 96 19
promjena 24 89
promjena 6 26
sef 7 44
promjena 39 94
promjena 45 66
promjena 27 94
sef 39 91
promjena 51 59
sef 48 91
sef 73 100
sef 86 87
sef 24 70
promjena 69 35
sef 26 85
promjena 12 79
promjena 45 95
promjena 97 92
sef 92 97
sef 98 98
sef 26 61
promjena 12 8
sef 62 65
sef 69 80
promjena 68 39
promjena 32 56
sef 73 81
promjena 85 55
sef 8 98
sef 70 99
sef 56 77
sef 3 61
promjena 50 31
promjena 39 8
sef 52 92
sef 32 68
promjena 20 75
sef 81 90
sef 55 55
sef 46 98
sef 1 57
promjena 98 48
promjena 51 50
promjena 13 54
sef 5 34
promjena 66 78
sef 10 25
sef 84 98
sef 74 76
sef 77 82
promjena 31 66
promjena 45 70
sef 17 43
sef 34 70
promjena 79 66
sef 12 65
promjena 20 37
sef 92 96
promjena 34 97
sef 89 98
sef 78 79
promjena 1 54
promjena 16 47
promjena 31 55
promjena 8 38
promjena 27 35
sef 14 97
sef 61 95
promjena 58 85
sef 16 59
promjena 18 43
sef 68 92
sef 96 99
sef 4 85
sef 59 60
promjena 31 24
promjena 74 3
sef 21 51
sef 81 84
promjena 21 33
promjena 4 76
promjena 35 80
sef 33 47
