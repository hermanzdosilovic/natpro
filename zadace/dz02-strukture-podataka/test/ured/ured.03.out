37
37
1
16
13
3
23
3
3
1
1
1
5
1
1
5
37
3
1
13
13
1
1
1
3
1
1
1
68
1
3
3
3
1
13
15
3
1
1
5
5
36
1
1
3
1
19
1
3
3
1
8
